packages/forms/src/validators.ts

```diff
@@ -168,7 +168,7 @@ export class Validators {
    * @returns A validator function that returns an error map with the
    * `min` property if the validation check fails, otherwise `null`.
    *
-   * @see {@link updateValueAndValidity()}
+   * @see [updateValueAndValidity()](/api/forms/updateValueAndValidity)
```

Comment:
```suggestion
   * @see [updateValueAndValidity()](/api/forms/AbstractControl#updateValueAndValidity)
```

It's a method on `AbstractControl`